test/framework/mock_stream.h

```diff
@@ -0,0 +1,149 @@
+/*
+ *
+ * Copyright 2017 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPCPP_TEST_MOCK_STREAM_H
+#define GRPCPP_TEST_MOCK_STREAM_H
+
+#include <stdint.h>
```

Comment:
This is a copy of https://github.com/grpc/grpc/blob/34e8e0a6400d8b529125a3b83ec1facf71acf99b/include/grpcpp/test/mock_stream.h .
I think it is because of `uint32_t` here: https://github.com/hyperledger/iroha/blob/83bf44f91448474207c9fc99a837fef83be44bc0/test/framework/mock_stream.h#L73

Reply:
So why we can't remove it?

Reply:
Because `uint32_t` is defined in stdint.h